### ISSUE TITLE
Always showing column via default behavior of columnGroupShow property leads to warning 

### ISSUE THREAD

**Issue by DFPy** [ag-grid/ag-grid#6305]
**I'm submitting a ...**
```
[] bug report
[] feature request
[x] support request
```

**Requesting Community Support**

The [column definitions documentation](https://www.ag-grid.com/vue-data-grid/column-groups/#showing--hiding-columns) states, that setting the `columnGroupShow` property to anything other than 'open' or 'closed' leads to the respective column always being shown. Using this functionality to always show a column by setting its columnGroupShow property to 'always' yields the warning `AG Grid: 'always' is not valid for columnGroupShow. Valid values are 'open', 'closed'` in the browser console.

This behavior occurred for the first time after migrating from Vue 2 to Vue 3. Is this a new, intended behavior, a transitory artifact of the compatibility build that will vanish once the migration to Vue 3 is complete or is this hinting at another issue?

 

**Please tell us about your environment:**
* **OS:** Windows 10 Enterprise

* **IDE:** IntelliJ Ultimate

* **AG Grid version:** 29.0.0 Community and Enterprise for Vue 3

* **Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/110.0

* **Language:** VueJS: Vue 3 using compatibility build in Vue 2 mode

**Comment by StephenCooper** [comment]
Thanks for highlighting this issue. I am going to update the docs to remove the reference to using any value. We will say that `undefined` | `null` should be used to always show the group.

**Comment by DFPy** [comment]
Thank you for the swift response!

**Comment by StephenCooper** [comment]
Our docs are now updated for 29.2 which contains the update mentioned above. 

https://www.ag-grid.com/vue-data-grid/column-groups/#showing--hiding-columns
